xml/Microsoft.Aspnet.Snapin/IDataObject.xml

```diff
@@ -98,10 +133,24 @@
         <Parameter Name="a" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="a">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="a">The address of the <see cref="T:System.Runtime.InteropServices.ComTypes.IEnumSTATDATA" /> receiving the interface pointer to the new enumerator object. There are no connections if this call to the EnumDAdvise method sets a to null (Nothing in Visual Basic).</param>
```

Comment:
formatting of a here would be helpful, since otherwise it looks like content has dropped out (\<paramref name="a" />)